lib/di/interceptor/app_interceptor.dart

```diff
@@ -2,25 +2,26 @@
 import 'dart:io';
 
 import 'package:dio/dio.dart';
+import 'package:flutter_survey/utils/storage/secure_storage.dart';
 
 const String _authorizationHeader = 'Authorization';
 
 class AppInterceptor extends Interceptor {
   final bool _requireAuthentication;
   final Dio _dio;
+  final SecureStorage _secureStorage;
 
-  AppInterceptor(
-    this._requireAuthentication,
-    this._dio,
-  );
+  AppInterceptor(this._requireAuthentication, this._dio, this._secureStorage);
```

Comment:
we should put params in multiple lines when there are more than 1 to improve readability.